### DISCUSSION TITLE
Store beatmap panel size choice

### DISCUSSION THREAD

**Discussion by Jeglerjeg** [ppy/osu#17420] [Ideas] [answered]
Currently the game resets your choice of beatmap panel size between every restart, it would be nice to have it persist

**Comment by bdach** [comment]
was aware of this when i was reimplementing, but wasn't sure how the setting should be exposed. web just stores the latest preference silently in localstorage without any way to configure it from anywhere else; i could do the same, or i could add an option in settings for that (but i would probably prefer not to).

also there's the other question whether it should apply only to the beatmap listing or globally. web-side it applies globally even though it is only changeable on the beatmap listing which is a bit weird.

**Reply by Jeglerjeg** [reply]
I think it would at the very least make sense for whatever choice you make in the beatmap listing to persist, whether or not there's also a separate option in settings for it. It's a bit weird for it only to be changeable in beatmap listings but applying to other parts of lazer, but I think it'd be even weirder sense for the display to be inconsistent depending on where you are.

**Reply by peppy** [reply]
I think this kind of setting is fine to persist but not be in the global settings.

**Comment by frenzibyte** [comment, marked as answer]
Has been resolved by #17615
